src/pluto/pluto.py

```diff
@@ -521,6 +521,40 @@ def create_expected(df, v1, v2):
                     st.markdown(f"* in {v2} but not in {v1}:")
                     st.write(in2not1)
 
+    def create_corrections(df):
+        def generate_trace(v1):
+            hovertemplate = "<b>%{x} - %{y}</b>"
+            return go.Scatter(
```

Comment:
A little confusing that this is called "Scatter" instead of "Line", is there a better named graph type we can use?